src/components/Produtos.js

```diff
@@ -0,0 +1,168 @@
+	width: 13vw;
+  height: 13vw;
+`
+const FiltrosSacola = styled.div`
+  width: 18vw;
+	grid-row-start: 1;
+	grid-row-end: 2;
+	grid-column-start: 5;
+	grid-column-end: 6;
+`
+
+class Produtos extends React.Component {
+
+  state = {
+    order: 'NOME',
+    produtos: [],
+    novoProduto: [],
+  }
+
+  componentDidMount() {
+    this.pegaProdutos()
+  }
+
+  
+  onChange = (event) => {
+      const escolha = event.target.value
+      this.setState({order: escolha})
+  }
+
+  orderArray = () => {
+    let newArray = []
+    switch (this.state.order) {
+      case "NOME":
+        newArray = this.state.produtos.sort((a,b) => a.name > b.name)
+        break;
+      case "CATEGORIA":
+        newArray = this.state.produtos.sort((a,b) => a.category > b.category)
+        break;
+      case "PRECO":
+        newArray = this.state.produtos.sort((a,b) => a.price - b.price)
+        break;
+      default:
+        newArray = this.state.produtos
+        break;
+    }
+    console.log(newArray)
+    return newArray
+  }
+
+  pegaProdutos = () => {
+		
+		axios
+			.get(
+        "https://us-central1-labenu-apis.cloudfunctions.net/eloFourOne/products"
+        )
+			.then((response) => {
+        this.setState({ produtos: response.data.products });
+			})
+			.catch((error) => {
+				alert("Opa!");
+			});
+	};
+
+  addToCart = (id) => {
+    const sacolaArray = this.state.produtos.filter((produto) => {
+      if (produto.id === id) {
+        return true
+      }
+    })
+    this.setState({novoProduto: this.state.novoProduto.push(sacolaArray)})
+    console.log("Depois:", this.state.novoProduto)
+    return this.state.novoProduto
+  }
+
+ /*  removeItem = (id) => {
+    const cartArray = this.state.productsArray.map((product) => {
+      if (product.id === id) {
+        const newProduct = { ...product, quantity: 0}
+        return newProduct
+      } else {
+        return product
+      }
+    })
+    this.setState({ productsArray: cartArray })
+  }
+ */
+
+  render() {
+    const arrayOrdernado = this.orderArray()
+    const arraySacola = this.addToCart()
+    const ListaDeProdutos = this.state.produtos.map((produto) => {
+      return (
+        <CardsProdutos
+          key={produto.id}
+          imagem={produto.photos}
+          nome={produto.name}
+          preco={produto.price}
+          parcelas={produto.installments}
+          metodo={produto.paymentMethod}
+          id={produto.id}
+          addToCart={this.addToCart}
+        />
+      );
+    });
+  
+    return (
+      <div>
+      <ProductsHeader>
+                <Label>
+                    Ordenar: 
+                    <select onChange = {this.onChange}>
+                        <option value={'NOME'}>Nome</option>
+                        <option value={'PRECO'}>Preço</option>
+                        <option value={'CATEGORIA'}>Categoria</option>
+                    </select>
+                </Label>
+                <p>Quantidade de Produtos: 3</p>
```

Comment:
Aqui seria mais interessante que a quantidade fosse definida de acordo com o tamanho do array de produtos de voces!
```suggestion
                <p>Quantidade de Produtos: {this.state.produtos.length}</p>
```